bezsmertnyi-stanislav/Readme.MD

```diff
@@ -0,0 +1,68 @@
+Flashscore Live Scores
+======================
+
+## Introduction
+
+The Flashscore Live Scores app provides real-time sports scores and updates from various sporting events worldwide. Stay up-to-date with the latest scores, fixtures, and results for your favorite sports and teams. The most up-to-date live scores, results, stats, news and other info from football, tennis, basketball and 30+ sports and more than 6000 competitions from all around the world. You can select your favourite matches, teams or players, and our notifications will let you know about every important moment of the match.
+
+![](https://i.imgur.com/NRPPEeA.jpg)
+
+## Installation
+
+* Download the Flashscore Live Scores app from [AppStore](https://apps.apple.com/app/id766443283?mt=8) or [Google Play Store](https://play.google.com/store/apps/details?id=eu.livesport.FlashScore_com)
+* Install the app. You'll find the downloaded app in "Notifications" or in the "Download" folder. Click on it, choose "Install" and confirm
+* Launch the app
+
+## Features
+
+* COVERAGE: Discover live scores for football, tennis, basketball, and over 30 other sports (including rugby, cricket, and more) within our application. We monitor 6000+ competitions from around the world, with a dedicated focus on over 1000+ football tournaments alone!
+
+* SPEED: You will immidiatly receive updates, synchronized with the live audience, about scored goals, red cards, a set or period concludes.
+
+* FAVOURITE TEAMS AND MATCHES: You can follow your favourite matches, teams and competitions.
+
+* NOTIFICATIONS AND ALERTS: You can select your preferred matches and await the timely notifications on your mobile device.
+
+* SYNCHRONIZATION: As a registered user, your personalized live scores data will be accessible across all your devices.
+
+* REAL-TIME COMMENTARY: Stay up-to-date with our live text commentary.
+
+* LINE-UPS AND HEAD-TO-HEAD RECORDS: We provide advanced access to line-ups and the historical records, enabling you to find out the past encounters between both teams.
+
+## Usage
+
+* ### How to register
+
+    * Go to settings by tapping on cogwheel icon
+    * Click on button "Log in" in user section
+    * On the registration screen choose the way how you want to register: with email, with facebook, with google
+    * If you choose with facebook or with google, just choose your account on facebook or gmail
+    * If you choose with email, you need to input email and come up with password
```

Comment:
Here's an example of how this can be edited:
1. Go to settings by selecting the cogwheel icon.
2. Click the "Log in" button in the User section.
3. On the registration screen, choose how you want to register: with email, Facebook, or Google.
	* To register with email, provide your email address and come up with a password.
	* To register with a Facebook or Google account, select the account to use.